### PR TITLE
GJ-135 add wallets quickstart guide to explain the flow

### PR DESCRIPTION
**Pull request by Tom-Goji**
Add a wallet quickstart guide to walk through the flow of adding a company, a wallet, registering some details and sending money from the wallet. Probably can be expanded upon but feels like a good starter for 10 :)

### CODE REVIEW

File: source/includes/_quickstart-wallets.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+## <em>Quickstart Wallets</em> 🚧🚧
+
+This guide gives a brief overview to how to do the following, using the wallet API:
+
+- Register parties
```
Comment (hoto) [inline source/includes/_quickstart-wallets.md]: Shouldn't be `Register companies`?

File: source/includes/_quickstart-wallets.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+- Create wallets
+- Registering a location outside the system to send money to
```
Comment (hoto) [inline source/includes/_quickstart-wallets.md]: Don't we usually reference "location" as "payment destination" in other places?

File: source/includes/_quickstart-wallets.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+All money held in wallets is eMoney and therefore might not be suitable for all use cases. Let us know if you have questions.
+
+- [Payments API](/#payments-managers)
+
+## Registering companies
+
+In our payments system we refer to all different types of entity as a Party. In the case of [registering a company](/#payments-manager-post-companies) you will receive a `partyId` in the response. This will look something like `COM~d28360c5-07a3-4d78-ade4-bddcdd8b5502` and is our reference to that entity which will be used to create wallets under that party.
+
+It's worth noting that money in wallets underneath this party will be held in their name. For example a if we registered Goji under the wallet API, money held in any wallet under that party would be owned by Goji. Likewise if we registered a Borrower in the P2P space, any money in wallets under that borrower is owned by the Borrower.
```
Comment (hoto) [inline source/includes/_quickstart-wallets.md]: Looks like a typo in `For example a if` or is this example `a` but then where is `b` 🤔

File: source/includes/_quickstart-wallets.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+Registering new parties is an asyncronous process therefore the party will not be ready for use straight away. You will receive a [webhook](/#webhooks-company_registration_update) when the party is enabled for use.
```
Comment (hoto) [inline source/includes/_quickstart-wallets.md]: Looks like a typo in `asynchronous`, should be `asynchronous`?
Same couple lines below.

File: source/includes/_quickstart-wallets.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
+## <em>Quickstart Wallets</em> 🚧🚧
+
+This guide gives a brief overview to how to do the following, using the wallet API:
```
Comment (hoto) [inline source/includes/_quickstart-wallets.md]: Grammarly says it should be `overview of how`

File: source/includes/_quickstart-wallets.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+- Register parties
+- Create wallets
+- Registering a location outside the system to send money to
+- Make a outbound payment
```
Comment (hoto) [inline source/includes/_quickstart-wallets.md]: `Make an outbound`

File: source/includes/_quickstart-wallets.md (hunk 4 above)
Comment (hoto) [reply source/includes/_quickstart-wallets.md]: We use the status of `REGISTERED` when the company is "enabled".
Should we either rename the status to `ENABLED` or use `registered` here to be consistent?
It says `registered` couple lines below.

File: source/includes/_quickstart-wallets.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+In our payments system we refer to all different types of entity as a Party. In the case of [registering a company](/#payments-manager-post-companies) you will receive a `partyId` in the response. This will look something like `COM~d28360c5-07a3-4d78-ade4-bddcdd8b5502` and is our reference to that entity which will be used to create wallets under that party.
+
+It's worth noting that money in wallets underneath this party will be held in their name. For example a if we registered Goji under the wallet API, money held in any wallet under that party would be owned by Goji. Likewise if we registered a Borrower in the P2P space, any money in wallets under that borrower is owned by the Borrower.
+
+Registering new parties is an asyncronous process therefore the party will not be ready for use straight away. You will receive a [webhook](/#webhooks-company_registration_update) when the party is enabled for use.
+
+## Creating wallets
+
+Once you've received the webhook from the previous stage letting you know the party is registered, we will be ready to create wallets for that Party.
```
Comment (hoto) [inline source/includes/_quickstart-wallets.md]: `we will be ready to create wallets for that Party`
Shouldn't this be `you will...`?
Or `wallets can be created for that Party`?

Also should the `Party` be capitalise here? We use lower case in other places.

File: source/includes/_quickstart-wallets.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+
+Wallets are created under the [Managing Wallets API](/#payments-manager-post-wallets). Creating a wallet is an asyncronous action due to slow responses from downstream providers. Once we create a wallet for a given party we'll receive a unique `id` for the wallet in the response. This is then used in any subsequent wallet related calls.
```
Comment (hoto) [inline source/includes/_quickstart-wallets.md]: Looks like `for a given party we'll receive` should be `for a given party you'll receive`

Also the `id` of the wallet is returned from the initial `POST` call to crate wallet.
And this paragraph kind of suggest they have to wait for it to be in the webhook?

I would maybe move the `Creating a wallet is an asyncronous action due to slow responses from downstream providers.` to the next paragraph about webhooks?

File: source/includes/_quickstart-wallets.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+
+## Registering locations outside the system to send money to
+
+In order to send money outside of the system we need to register the recipient bank details and run AML checks on the them. You can use the [bank destination API](/#payments-manager-post-platformapi-bankaccountdetails) to add new accounts and these can be registered once for use across multiple party wallets.
+
+This is also an asyncronous action so you'll receive a [webhook](/#webhooks-bank_account_details_status_changed) once they've passed AML and are ready to use.
+
+## Make a outbound payment
+
+Once we have our wallet created and the destination account enabled, we're ready to send money. To do this we can use the [wallet payment API](/#payments-manager-post-wallets-id-payment). Please note that we currently do not support sending in multiple currencies via this API. If this is a problem please get in touch.
```
Comment (hoto) [inline source/includes/_quickstart-wallets.md]: Should the `Once we have our` be `Once you have a wallet`?
This guide refers to "our system" which suggest "we" are "goji" but this paragraph uses "we" as "them".

File: source/includes/_quickstart-wallets.md [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+
+Once you've received the webhook from the previous stage letting you know the party is registered, we will be ready to create wallets for that Party.
+
+Wallets are created under the [Managing Wallets API](/#payments-manager-post-wallets). Creating a wallet is an asyncronous action due to slow responses from downstream providers. Once we create a wallet for a given party we'll receive a unique `id` for the wallet in the response. This is then used in any subsequent wallet related calls.
+
+You'll receive a [webhook](/#webhooks-wallet_created) when the wallet is enabled and ready for use.
+
+## Registering locations outside the system to send money to
+
+In order to send money outside of the system we need to register the recipient bank details and run AML checks on the them. You can use the [bank destination API](/#payments-manager-post-platformapi-bankaccountdetails) to add new accounts and these can be registered once for use across multiple party wallets.
```
Comment (hoto) [inline source/includes/_quickstart-wallets.md]: Grammarly points out `AML checks on the them` should be `AML checks on them`

File: source/includes/_quickstart-wallets.md [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+
+This is also an asyncronous action so you'll receive a [webhook](/#webhooks-bank_account_details_status_changed) once they've passed AML and are ready to use.
+
+## Make a outbound payment
```
Comment (hoto) [inline source/includes/_quickstart-wallets.md]: `Make an outbound`

File: source/includes/_quickstart-wallets.md (hunk 1 above)
Comment (Tom-Goji) [reply source/includes/_quickstart-wallets.md]: In this case we ask them to provide the `owningPartyId` so I think it's clearer to say they give us a company and we respond with a party.

File: source/includes/_quickstart-wallets.md (hunk 3 above)
Comment (Tom-Goji) [reply source/includes/_quickstart-wallets.md]: Ugh lack of coffee 😬

File: source/includes/_quickstart-wallets.md (hunk 4 above)
Comment (Tom-Goji) [reply source/includes/_quickstart-wallets.md]: And again although I didn't use `registered` but `ready` instead 👍

File: source/includes/_quickstart-wallets.md (hunk 7 above)
Comment (Tom-Goji) [reply source/includes/_quickstart-wallets.md]: Ha clearly drifted into siding with the client a bit too much. Too much rushing trying to get it out for Nester...

**Pull request merged**